src/FormMigration/FormMetaDecorator.php

```diff
@@ -684,4 +683,86 @@ private function isDefaultFund(int $fundId): bool
 
         return $fund->id === $fundId;
     }
+
+    /**
+     * @unreleased
+     *
+     * @return string 'global', 'enabled', 'disabled'
+     */
+    public function getGiftAidStatus(): string
+    {
+        return $this->getMeta('give_gift_aid_enable_disable');
+    }
+
+    /**
+     * @unreleased
+     */
+    public function getGiftAidTitle(): string
+    {
+        return $this->getMeta('give_gift_aid_fieldset_title');
+    }
+
+    /**
+     * @unreleased
+     */
+    public function getGiftAidDescription(): string
+    {
+        return $this->getMeta('give_gift_aid_explanation_content');
+    }
+
+    /**
+     * @unreleased
+     */
+    public function getGiftAidLongExplanationEnabled(): bool
+    {
+        return $this->getMeta('give_gift_aid_long_explanation_enable_disable');
+    }
+
+    /**
+     * @unreleased
+     */
+    public function getGiftAidLinkText(): string
+    {
+        return __('Tell me more', 'give-gift-aid');
+    }
+
+    /**
+     * @unreleased
+     */
+    public function getGiftAidModalHeader(): string
+    {
+        return __('What is Gift Aid?', 'give-gift-aid');
```

Comment:
If these are static strings (possibly new settings?) and not meta of the form then they can be added directly to the migration step process method.

Reply:
Refactored on https://github.com/impress-org/givewp/pull/7186/commits/26e3e881070eb1f484126a2ebd149a8ee4edefe7